cms/templates/cms/home_page.html

```diff
@@ -117,14 +117,16 @@
       <div class="row">
         <div class="col-md-8">
           <img src="static/images/logo-mit-white.png" alt="">
-          <div class="footer-links"><a href="add" target="_blank">About Micromasters</a> <a href="https://www.edx.org/" target="_blank">edX</a> <a href="https://odl.mit.edu/" target="_blank">Office of Digital Learning</a>
+          <div class="footer-links"><a href="https://www.edx.org/" target="_blank">edX</a> <a href="https://odl.mit.edu/" target="_blank">Office of Digital Learning</a>
           </div>
           <div class="footer-address">
             Massachusetts Institute of Technology<br> Cambridge, MA 02139
           </div>
         </div>
         <div class="col-md-4 text-right">
-          <div class="footer-cta"></div>
+          <div class="footer-cta"><button href="https://giving.mit.edu/" target="_blank"
```

Comment:
the correct URL is https://giving.mit.edu/explore/campus-student-life/digital-learning

per https://github.com/mitodl/mga/issues/5#issuecomment-221979829